src/main/webapp/WEB-INF/classes/hibernate.cfg.xml

```diff
@@ -37,5 +37,7 @@
 		<mapping class="com.strandls.observation.pojo.Recommendation" />
 		<mapping
 			class="com.strandls.observation.pojo.RecommendationVote" />
+		<mapping class="com.strandls.observation.pojo.DataTable"/>
```

Comment:
remove the mapping from here, not required, initial we use to do that, but now its not required